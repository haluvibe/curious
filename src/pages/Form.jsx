import React from "react";

export default () => 
  <div>
    <h2 className="title">Form Page</h2>
  </div>
;
