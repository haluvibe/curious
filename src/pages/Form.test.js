import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import Form from "./Form";

test("Form component should render as expected", () => {
    const component = shallow(<Form />),
        tree = toJson(component);
    expect(tree).toMatchSnapshot();
    expect(component.find(".title").length).toBe(1);
});
