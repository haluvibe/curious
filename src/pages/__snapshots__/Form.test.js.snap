// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component should render as expected 1`] = `
<div>
  <h2
    className="title"
  >
    Form Page
  </h2>
</div>
`;
